source/extensions/tracers/opencensus/BUILD

```diff
@@ -25,6 +25,7 @@ envoy_cc_library(
     name = "opencensus_tracer_impl",
     srcs = ["opencensus_tracer_impl.cc"],
     hdrs = ["opencensus_tracer_impl.h"],
+    copts = ["-Wno-unused-parameter"],
```

Comment:
Can you add this parameter to OpenCensus build to avoid doing this in future? It's fine as a temporary workaround, but please add a TODO to remove.

Reply:
The warnings come from googleapis code, which is being included by code that lives in the Envoy repo. I don't think adding `-Wno-unused-parameter` to `@io_opencensus_cpp//opencensus/exporters/trace/stackdriver:stackdriver_exporter` would resolve this problem. :(

Please let me know if I'm missing something here.